Makefile

```diff
@@ -64,6 +64,11 @@ define generate_k8s_api_deepcopy
 	$(call generate_k8s_api,deepcopy,$(1),$(2))
 endef
 
+define NL
```

Comment:
Please add a comment that describes what this is doing and why with a reference to the issue.

Reply:
Done.